app/views/licence_transaction/start.html.erb

```diff
@@ -5,6 +5,9 @@
   publication: publication,
   edition: @edition,
 } do %>
+
+  <%= render "govuk_publishing_components/components/lead-paragraph", text: publication.summary %>
```

Comment:
I think this needs to be `publication.description`

Reply:
I don't think the "start" template is rendered for every type of licence (local authority / external), so "summary" also needs to be added to the `continues_on` and possibly the `authority` templates.

Reply:
Nice. Hmm, the authority page also displays the overview (or 'body') so may be worth adding to the authority view too? E.g https://www.integration.publishing.service.gov.uk/find-licences/auction-premises-registration/rotherham. It feels a bit odd that the same information is repeated on the pre and post postcode views but 🤷  

The multiple authority view doesn't return the 'body' from looking at the code so I think we're fine excluding the description from that view (https://github.com/alphagov/frontend/blob/29d7ecbdb40202c59f423d7a21ccc0bc07b38027/app/views/licence_transaction/multiple_authorities.html.erb#L1). I don't have an example of this page though.

Probably worth running it by Alice to double check 👍

Reply:
I've asked Alice. The multiple authorities is just a disambiguator for split postcodes, so I think that's fine.

Reply:
> It feels a bit odd that the same information is repeated on the pre and post postcode views

That is really odd. I wonder how many people actually scroll down to read the information on the postcode lookup page compared to the authority page.